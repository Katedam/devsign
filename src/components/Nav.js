import React from 'react';
import SearchForm from './SearchForm';
import { ListItem, StyledLink } from '../styles/mainStyle';
import styled from 'styled-components';

const NavBar = styled.nav `
  display: flex;
  align-items: space-between;
  background-color: #f3d8c4;
`;

const HideNav = styled(NavBar) `
  @media (max-width: 700px) {
    display: none;
  }
`;

const List = styled.ul `
  width: 25%;
  padding: 5px;
  margin: 10px;
`;

const NavLi = styled(ListItem) `
  color: #c16834;
  padding: 0 20px;
`;

function Nav() {
  return (
    <HideNav>
      <List>
        <StyledLink to="/home"><NavLi>Home</NavLi></StyledLink>
        <StyledLink to="/profile"><NavLi>Profile</NavLi></StyledLink>
        <StyledLink to="/followers"><NavLi>Followers</NavLi></StyledLink>
      </List>
      <SearchForm />
    </HideNav>
  );
}

export default Nav;
