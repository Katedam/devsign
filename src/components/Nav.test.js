import React from 'react';
import Nav from './Nav';
import { shallow } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';

describe('Nav component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(
      <MemoryRouter>
        <Nav />
      </MemoryRouter>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
