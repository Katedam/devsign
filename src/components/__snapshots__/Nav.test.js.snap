// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component matches a snapshot 1`] = `
<nav
  className="sc-kGXeez kHJuoe"
>
  <ul
    className="sc-kpOJdX ifvqSH"
  >
    <li
      className="sc-gZMcBi sc-ckVGcZ fJmOAI"
    >
      <a
        className="sc-bdVaJa sc-hMqMXs hDuBxd"
        href="/home"
        onClick={[Function]}
      >
        Chirp
      </a>
    </li>
    <li
      className="sc-gZMcBi sc-dxgOiQ btSlKp"
    >
      <a
        className="sc-bdVaJa sc-eNQAEJ gsdyPY"
        href="/home"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="sc-gZMcBi sc-dxgOiQ btSlKp"
    >
      <a
        className="sc-bdVaJa sc-eNQAEJ gsdyPY"
        href="/profile"
        onClick={[Function]}
      >
        Profile
      </a>
    </li>
  </ul>
  <a
    className="sc-bdVaJa cVFclJ"
    href="/profile"
    onClick={[Function]}
  >
    <img
      alt="profile pic"
      className="sc-jKJlTe fhvuZn"
      src="https://joeschmoe.io/api/v1/jeane"
    />
  </a>
  <form
    className="sc-ifAKCX sc-cSHVUG bPDerF"
  >
    <label
      className="sc-kgoBCf iOxteu"
      htmlFor="search"
    >
      Search
    </label>
    <input
      className="sc-bZQynM sc-kAzzGY hecALk"
      id="search"
      readOnly={true}
      type="text"
      value=""
    />
    <label
      className="sc-kgoBCf iOxteu"
      htmlFor="searchButton"
    />
    <button
      className="sc-gzVnrw sc-chPdSV eNmjaL"
      id="searchButton"
    >
      Search
    </button>
  </form>
</nav>
`;
